hw/dv/dpi/usbdpi/usb_monitor.c

```diff
@@ -2,6 +2,11 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+// The code below uses setlinebuf, which comes from stdio.h. With glibc, it is
+// only provided if the _DEFAULT_SOURCE feature test macro is defined (because
+// it came from BSD in the first place, so needs pulling in explicitly).
+#define _DEFAULT_SOURCE
```

Comment:
Perhaps replace `setlinebuf` with `setvbuf(mon->file, NULL, _IOLBF, 0)` instead?

Reply:
The other item looks like it's undefined behaviour :(

`cfmakeraw` modifies an initialized termios struct, it should not be used to create something new.

Reply:
Oh dear! I'll add a follow-up commit to this PR that sorts that out.